docs/README.md

```diff
@@ -38,13 +38,24 @@ Report consist of 2 parts:
 ### Manifests
 
 List of supported configuration to manifest translations:
-  * Crio
-  * Cluster Resources Quotas and Resources Quotas
-  * Image
-  * OAuth
-  * Project
+  * CRI-O
+    * If defined in OCP3's cluster, the CRI-O configuration defined in 'crio.conf' is ported to a machineconfiguration.openshift.io resource and saved under '100_CPMA-crio-config.yaml'.
+  * Cluster Resources Quotas
+    * Every quota defined at the cluster level is exported into equivalent OCP4 CR file. The file name is in the form: '100_CPMA-cluster-quota-resource-<ClusterQuota name>.yaml'
+  * Resources Quotas
+    * Within a given namespace/project, each quota resource is exported into equivalent OCP4 CR file in the form of '100_CPMA-<Namespace>-resource-quota-<ResourceQuota Name>.yaml'
+  * Image Configuration
+    * An image.config.openshift.io resource, saved under file name '100_CPMA-cluster-config-image.yaml` is created from the following sources:
```

Comment:
Are you sure there is a right quote - `?

Reply:
Thanks!